src/python/WMComponent/WorkflowUpdater/SiteListUpdater.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python
+"""
+File       : SiteListUpdater
```

Comment:
Just to keep things more consistent across components, perhaps naming this as `SiteListPoller` would be better.

Reply:
done